analyzers/src/SonarAnalyzer.CSharp/Rules/ReturnEmptyCollectionInsteadOfNull.cs

```diff
@@ -68,41 +68,32 @@ void ReportIfAny(List<Location> nullOrDefaultLiterals)
         }
     }
 
+    private static BlockSyntax GetBody(SyntaxNode node) =>
```

Comment:
Can't we remove this method and use the SyntaxExtensions?

Reply:
Discussed ofline, this always returns the body of the `get` accessor. We would have to return an array instead of a single `BlockSyntax`.